editor.planx.uk/src/pages/Team.tsx

```diff
@@ -27,46 +29,79 @@ import { FlowSummary } from "./FlowEditor/lib/store/editor";
 import { formatLastEditMessage } from "./FlowEditor/utils";
 
 const DashboardList = styled("ul")(({ theme }) => ({
-  padding: theme.spacing(0, 0, 3),
-  borderBottom: "1px solid #fff",
+  padding: theme.spacing(3, 0),
   margin: 0,
+  display: "grid",
+  gridAutoRows: "1fr",
+  gridTemplateColumns: "repeat(1, 1fr)",
+  gridGap: theme.spacing(2),
+  [theme.breakpoints.up("md")]: {
+    gridTemplateColumns: "repeat(2, 1fr)",
+  },
+  [theme.breakpoints.up("lg")]: {
+    gridTemplateColumns: "repeat(3, 1fr)",
+  },
```

Comment:
I love CSS grid and I think we don't use it half enough! This works really well for the layout and feels nice and responsive 👏